fatal/type/impl/apply.h

```diff
@@ -11,18 +11,125 @@
 #define FATAL_INCLUDE_fatal_type_impl_apply_h
 
 namespace fatal {
-namespace i_A {
+namespace i_ap {
 
-template <template <typename...> class, typename...> struct ls;
+template <template <typename...> class T, typename... Args>
+struct a { using type = T<Args...>; };
+
+template <template <typename> class T, typename U>
+struct a<T, U> { using type = T<U>; };
```

Comment:
Is this not generalizable?

Reply:
@yfeldblum: If you pass a non-variadic template as T you can't expand the arguments.
Sucks balls, but this is the workaround that deprecated apply used.